homeassistant/components/modbus/modbus.py

```diff
@@ -278,6 +278,8 @@ def __init__(self, hass: HomeAssistant, client_config: dict[str, Any]) -> None:
             _LOGGER.warning(
                 "`retries`: is deprecated and will be removed in version 2024.7"
             )
+        else:
+            client_config[CONF_RETRIES] = 3
```

Comment:
If we don't use retries anymore, is there a real need to set it?

Reply:
who said we do not use it internally ???

We use it internally until I later bump the library, we just do not want people to configure it.

Reply:
Oooh, never mind. I misread the "the maximum retries is now 3" as "the retries is now always set to 3"